README.md

```diff
@@ -52,4 +52,4 @@ details on submitting patches and the contribution workflow.
 ## Membership
 We encourage all contributors to become members. Learn more about requirements and responsibilities of membership in our [Community Membership doc](./community-membership.md).
 
-If you have made contributions that meet the requirements of becoming Volcano member, [simply file an issue](https://github.com/volcano-sh/community/issues/new) to apply.
\ No newline at end of file
+If you have made contributions that meet the requirements of becoming Volcano member, [simply file an issue](https://github.com/volcano-sh/community/issues/new) to apply.
```

Comment:
A little doubt that what's the diff here?

Reply:
My editor was adding a \n at the end of the file. Thanks for noticing it! Just removed.